introduction.md

```diff
@@ -1,3 +1,2 @@
-foobar
-
-I like the JavaScript code path
\ No newline at end of file
+* foobar
+* I like the JavaScript code path
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?